### PR TITLE
Fix/retry logic

### PR DESCRIPTION
**Pull request by gintarasm**
Retry logic causes locking issues. This will be fixed with async writer

### CODE REVIEW

**Review by AlgMi — DISMISSED** [review]

File: src/test/java/com/vinted/flink/bigquery/BigQueryBufferedSinkTest.java [hunk 1]
```diff
@@ -102,7 +102,7 @@ private Function<StreamExecutionEnvironment, DataStream<Rows<String>>> pipeline(
     private Function<StreamExecutionEnvironment, DataStreamSink<Rows<String>>> withBigQuerySink(MockJsonClientProvider<String> mockClientProvider, Function<StreamExecutionEnvironment, DataStream<Rows<String>>> pipeline) {
         var sink = BigQueryStreamSink.<String>newBuilder()
                 .withClientProvider(mockClientProvider)
-                .withDeliveryGuarantee(DeliveryGuarantee.AT_LEAST_ONCE)
+                .withDeliveryGuarantee(DeliveryGuarantee.EXACTLY_ONCE)
```
Comment (s-gelazevicius) [inline src/test/java/com/vinted/flink/bigquery/BigQueryBufferedSinkTest.java:105]: why did you changed it ?

Comment (s-gelazevicius) [reply src/test/java/com/vinted/flink/bigquery/BigQueryBufferedSinkTest.java:105]: oh, it's beffered test, ignore

Comment (gintarasm) [reply src/test/java/com/vinted/flink/bigquery/BigQueryBufferedSinkTest.java:105]: Cuase it is buffered sync in should have exactly once set

Comment (gintarasm) [reply src/test/java/com/vinted/flink/bigquery/BigQueryBufferedSinkTest.java:105]: it was an issue with the test

**Review by s-gelazevicius — APPROVED** [review]

**Pull request merged**